src/System.Management.Automation/security/SecuritySupport.cs

```diff
@@ -1253,31 +1245,33 @@ private void ResolveFromPath(SessionState sessionState, ResolutionPurpose purpos
         private void ResolveFromStoreById(ResolutionPurpose purpose, out ErrorRecord error)
         {
             error = null;
-            
+                        
```

Comment:
@mikeTWC1984 Please remove extra char. 
Nit: We have .editorconfig - please use these preferred editor configs from the file.

Reply:
Maybe we should add ps1 files to that config?

Reply:
They are there.